package.json

```diff
@@ -379,7 +379,8 @@
     "swc-loader": "^0.2.6",
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33279&commitId=58b5f6ca600852cc729176e8260175415fea8d6d&env=prod&clientType=FLUENTUI)


  <details><summary><b>Callout</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.Bottom Right Edge RTL.chromium.png|1593| Changed|
|Callout.Bottom auto edge.chromium.png|2309| Changed|
|Callout.Rendering callout attached to a rectangle.chromium.png|1493| Changed|
|Callout.Gap space 25.chromium.png|2309| Changed|
|Callout.Right center.chromium.png|2396| Changed|
|Callout.Right bottom edge.chromium.png|3005| Changed|
|Callout.Left bottom edge.chromium.png|3022| Changed|
|Callout.Right top edge.chromium.png|1127| Changed|

</details>


  <details><summary><b>react-charting-AreaChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-AreaChart.Custom Accessibility.chromium.png|11| Changed|

</details>


  <details><summary><b>react-charting-LineChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-LineChart.Gaps.chromium.png|1| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>